project-koku/koku
# Adding local OCI downloader flow

Fixes [COST-2384](https://issues.redhat.com/browse/COST-2384)

Changes proposed in this PR:
* Adding local OCI downloader flow
* Adding models for OCI processing
* Adding processing files
* Adding SQL for processing

Testing Instructions:
1. Add some local csv's to local download dir
2. Add oci-local provider
3. See oci_cost_line_items(_daily) and oci_usage_line_items(_daily) created and populated
4. See summary tables populated in postgres

---
Additional Context:
